pages/shared/data/tools.yaml

```diff
@@ -316,3 +316,13 @@ providers:
       fit: contain
     formats:
       - email
+  - name: Mailgun
+    url: https://mailgun.com/
+    type: Email Service
+    image:
+      src: /static/img/tools/mailgun.png
```

Comment:
This image doesn't exist yet

Reply:
Hi Sebil, thanks for your feedback. I have uploaded the official logo to that path in my branch. I also updated the dimensions to match the uploaded image, and updated the url.